packages/@lightningjs/ui-components/src/components/TabBar/Tab.d.ts

```diff
@@ -29,6 +29,7 @@ type TabStyle = SurfaceStyle & {
   contentColor: Color;
   iconMarginRight: number;
   iconSize: number;
+  /** @deprecated */
   noTitlePaddingX: number;
   paddingX: number;
```

Comment:
You'll still need `paddingXNoTitle` here so that it is made available to TypeScript devs. The `/** @deprecated */` for `noTitlePaddingX` will let devs know that it's no longer supported, and the `aliasStyles` you set up in Tab.js will allow the old style to still function until devs update their code, but you'll need `paddingXNoTitle` in the TypeScript definition so that devs can use the new style you set up.